ci-operator/step-registry/azure/provision/service-principal/minimal-permission/azure-provision-service-principal-minimal-permission-commands.sh

```diff
@@ -192,6 +192,127 @@ required_permissions="""
 \"Microsoft.Storage/storageAccounts/listKeys/action\"
 """
 
+if [[ "${ENABLE_MIN_PERMISSOIN_FOR_UPI}" == "true" ]]; then
```

Comment:
It is possible to get the permission list by `required_permissions` + the difference between IPI and UPI?

Reply:
updated